### PR TITLE
Add scripts to format YAML files

### PR DESCRIPTION
**Pull request by ixdy**
This first version *should* fail Travis, since I haven't formatted the files yet. I'll do that in a followup commit on this PR.

### CODE REVIEW

**Comment by spxtr** [comment]
I think you need to switch to [this](https://docs.travis-ci.com/user/trusty-ci-environment/) to use multiple languages.

File: .travis.yml [hunk 1]
```diff
@@ -9,14 +9,21 @@ services:
     - docker
 before_install:
     - docker pull gcr.io/google_containers/kubekins-job-builder:4
+# Install Go
+    - bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
```
Comment (spxtr) [inline .travis.yml]: beautiful

`--retry`?

File: gubernator/app.yaml [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 api_version: 1
-threadsafe: yes
-
+application: k8s-gubernator
 handlers:
-- url: /favicon\.ico
+- secure: always
   static_files: static/favicon.ico
   upload: static/favicon\.ico
+  url: /favicon\.ico
```
Comment (ixdy) [inline gubernator/app.yaml]: I'm not sure what happened in this file, but this looks wrong.

Comment (spxtr) [inline gubernator/app.yaml]: looks like it just alphabetized it.

**Comment by ixdy** [comment]
hm, this is definitely not going to work as-is.

Our use of anchors/aliases in the JJB configs is causing a host of problems.

Also need to figure out how to preserve comments.

**Comment by spxtr** [comment]
I'm a fan of dropping the anchors/aliases. Need to keep the comments, though.

File: .travis.yml [hunk 3]
```diff
@@ -9,6 +9,11 @@ services:
     - docker
 before_install:
     - docker pull gcr.io/google_containers/kubekins-job-builder:4
+# Install Go
```
Comment (rmmh) [inline .travis.yml]: Travis has built-in support for installing Go versions, just like Python.

    go: 
        - 1.6.2

**Comment by ixdy** [comment]
Without having some sort of yamlfmt that preserves comments I think auto-formatting YAML is infeasible. Closing this PR.

**Pull request closed without merging**